src/test/scala/edu/berkeley/cs/rise/opaque/SharedSparkSessions.scala

```diff
@@ -43,5 +44,6 @@ trait MultiplePartitionSparkSession {
       "spark.jars",
       "target/scala-2.12/opaque_2.12-0.1.jar,target/scala-2.12/opaque_2.12-0.1-tests.jar"
     )
+    .config("opaque.testing.enableSharedKey", true)
```

Comment:
I think this should be `spark.opaque.testing.enableSharedKey`, otherwise I couldn't get the Spark shell to work.

Reply:
What Spark shell? `build/sbt console`?

Reply:
I tried `spark-shell --jars ${OPAQUE_HOME}/target/scala-2.12/opaque_2.12-0.1.jar`

Reply:
Strange, I just tested that same command and it worked fine for me.

I added a line in the build to fix `build/sbt console`, maybe try again and let me know if it's still failing? Regardless, I'm going to go ahead and merge this in so we can fix the quickstart ASAP. We can discuss this further and I'll issue a new PR for a fix if needed.